app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```

Comment:
- You can use the same suggestion I made in post: `after_save :update_comments_counter`